ner/src/main/java/edu/illinois/cs/cogcomp/ner/NERAnnotator.java

```diff
@@ -100,20 +106,37 @@ public void initialize(ResourceManager nerRm) {
         ParametersForLbjCode.currentParameters.forceNewSentenceOnLineBreaks = false;
         Parameters.readConfigAndLoadExternalData(nerRm);
 
-         NETaggerLevel1 tagger1 =
-                new NETaggerLevel1(ParametersForLbjCode.currentParameters.pathToModelFile
-                        + ".level1", ParametersForLbjCode.currentParameters.pathToModelFile
-                        + ".level1.lex");
-
-        NETaggerLevel2 tagger2 = null;
-        if (ParametersForLbjCode.currentParameters.featuresToUse.containsKey("PredictionsLevel1")) {
-            tagger2 =
-                    new NETaggerLevel2(ParametersForLbjCode.currentParameters.pathToModelFile
-                            + ".level2", ParametersForLbjCode.currentParameters.pathToModelFile
-                            + ".level2.lex");
+        String dataset;
+        if(viewName.toLowerCase().contains("conll")) {
+            dataset = "enron-conll";
+        }
+        else {
+            dataset = "ontonotes";
+        }
+
+        String data_split = nerRm.getString(NerBaseConfigurator.TRAINED_ON);
+
+        try {
+            Datastore ds = new Datastore(new ResourceConfigurator().getDefaultConfig());
```

Comment:
Shouldn't this use the ResourceManager argument provided to the constructor to overwrite any properties the user wants to configure?

Reply:
Good point. Updated it.